cli/msg.fbs

```diff
@@ -577,4 +590,30 @@ table Seek {
 
 table GetRandomValues {}
 
+table DynamicLibLoad {
+  filename: string;
+}
+
+table DynamicLibLoadRes {
+  lib_id: uint32;
+}
+
+table DynamicLibFnLoad {
+  lib_id: uint32;
+  name: string;
+}
+
+table DynamicLibFnLoadRes {
+  fn_id: uint32;
+}
+
+table DynamicLibFnCall {
+  fn_id: uint32;
+  data: [ubyte];
+}
+
+table DynamicLibFnCallRes {
+  data: [ubyte];
```

Comment:
Also, use the deno ops to transfer data back- not through flatbuffers.

Reply:
Currently allowing for both data + zero copy. Any reason not to?